subprojects/maven/src/integTest/groovy/org/gradle/api/publish/maven/AbstractMavenPublishJavaIntegTest.groovy

```diff
@@ -670,6 +661,7 @@ Maven publication 'maven' pom metadata warnings (silence with 'suppressPomMetada
     @ToBeFixedForConfigurationCache
     def "can publish java-library with capability requests"() {
         given:
+        javaLibrary(mavenRepo.module("org.test", "foo", "1.0")).hasPackaging('pom').dependencyConstraint(mavenRepo.module('org.test', 'foo', '1.0')).withModuleMetadata().publish()
```

Comment:
💄 Now that's a very long line!

Reply:
Indeed. I will revert this change though, as it did not help us fix the issue (I still don't know how to set up things for this and some other test cases)